### PR TITLE
update to Juttle 0.5

### PR DESCRIPTION
**Pull request by davidvgalbraith**
@VladVega @demmer

### CODE REVIEW

File: lib/read.js [hunk 1]
```diff
@@ -24,7 +32,7 @@ class ReadS3 extends AdapterRead {
             .then(function(points) {
                 return {
                     points: points,
-                    readEnd: to
```
Comment (VladVega) [inline lib/read.js]: Nice - this must have recently gone into the code.

File: test/s3-adapter.spec.js [hunk 2]
```diff
@@ -4,25 +4,25 @@ var util = require('util');
...
 var client = Promise.promisifyAll(new AWS.S3(config));
 
-var S3Adapter = require('../lib')(config);
-Juttle.adapters.register(S3Adapter.name, S3Adapter);
 var test_bucket = 's3.adapter.test' + Math.random();
 
-describe('s3 source', function() {
-    this.timeout(300000);
-
+juttle_test_utils.withAdapterAPI(function() {
```
Comment (VladVega) [inline test/s3-adapter.spec.js]: curious - why do you need this here?

**Comment by VladVega** [comment]
+1

File: test/s3-adapter.spec.js (hunk 2 above)
Comment (davidvgalbraith) [inline test/s3-adapter.spec.js]: I suppose I don't need it. Seems more future-proof, though.

**Pull request merged**